src/main/java/duke/tools/SessionManager.java

```diff
@@ -0,0 +1,51 @@
+package duke.tools;
+
+import java.util.ArrayList;
+
+import duke.exceptions.DukeException;
+
+/**
+ * This class manages the objects created for the session.
+ */
+public class SessionManager {
+
+    /** The tasks stored by the user. */
+    private static TaskList taskList;
+    /** The storage to manage data files. */
+    private static Storage storage;
+
+    /**
+     * Private constructor to prevent object creation.
+     */
+    private SessionManager() {}
+
+    /**
+     * Initialize the SessionManager with the TaskList and Storage for the session.
```

Comment:
```suggestion
     * Initializes the SessionManager with the TaskList and Storage for the session.
```
Perhaps Intialize can better be worded as Initializes?